web-react-ts/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "web-react-ts",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@apollo/client": "^3.1.2",
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/icons": "^4.9.1",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "@types/jest": "^24.0.0",
```

Comment:
All these types libs can move to `-D` keep the deps clean

Reply:
Create react app doesn't do this distinction anymore, because theses apps generate bundles and don't have a runtime.
Read this thread and the answer from Dan Abramov: https://stackoverflow.com/questions/44868453/create-react-app-install-devdepencies-in-dependencies-section
However, I will change it tob be consistent with the other projects :)